x-pack/legacy/plugins/apm/public/components/shared/EuiTabLink.tsx

```diff
@@ -3,18 +3,27 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
+import React from 'react';
+import cls from 'classnames';
 import styled from 'styled-components';
-import { EuiTab } from '@elastic/eui';
 import theme from '@elastic/eui/dist/eui_theme_light.json';
 import { px, unit } from '../../style/variables';
 
+// TODO: replace this component with EUITab w/ a href prop
+// as soon as EUI is upgraded to 13.8.1
+// see https://github.com/elastic/eui/pull/2275
```

Comment:
`master` is already eui@13.8.1, `7.x` is on an older version. not sure why, but it should be possible to use EUITab in `master` right now.

Reply:
Yeah I noticed they merged it earlier today. I'll merge this fix in, and backport it, then see if we can drop this component completely with the new version of EUI being in master.

Reply:
Actually, we've made this a bit hard for ourselves because we use components as links, so there's no easy way for us right now to use `href` (or any other string-based approaches). Hmm.. I need to think about it for a bit.